adminutils/adminutils.py

```diff
@@ -183,6 +183,62 @@ async def emoji_add(self, ctx, name: str, url: str, *roles: discord.Role):
         else:
             await ctx.tick()
 
+    @emoji.command(name="steal")
+    async def emote_steal(self, ctx, name: str, message_id: int, *roles: discord.Role):
+        """
+        Steal an emoji from a message ID and add it to your server.
+        Role lock the emoji if you wish (use double quotes if role name has spaces)
+
+        Examples:
+            `[p]emoji steal thisReallyCoolName 743291269471797248`
+            `[p]emoji steal thisReallyCoolName 743291269471797248 RoleName`
+        """
+        channel = ctx.channel
+        message = await channel.fetch_message(message_id)
+        dig_up_emoji_check = message.content.split(":")[-3]
+        check_animated = dig_up_emoji_check.endswith("<a")
+        dig_up_emoji = message.content.split(":")[-1]
+        emoji = "".join(x for x in dig_up_emoji if x.isdigit())
+        check_animated = dig_up_emoji_check.endswith("<a")
+        if check_animated is True:
+            format_image = "gif"
+        else:
+            format_image = "png"
+        url = f"https://cdn.discordapp.com/emojis/{emoji}.{format_image}"
+        async with aiohttp.ClientSession() as session:
+            async with session.get(url) as r:
+                data = await r.read()
```

Comment:
```suggestion
        async with self.session.get(url) as r:
            data = await r.read()
```

Cog already have a session for that.

This also will throw an `check your logs` error if emoji is unable to unload, this better to be handled too. Look at how `emoji_add` handles that